server/fixtures/data.js

```diff
@@ -0,0 +1 @@
+export default [];
```

Comment:
What is this file doing here?. It is not exporting anything and it is named to generically.